lab2.js

```diff
@@ -39,31 +39,60 @@ function assert(expression, failureMessage) {
         1          |       2/hour
         2          |       3/hour
         3          |       4/hour
+*/
 
- TODO: First, make a constructor function, called Blob, that makes blobs.
+function Blob(blobName, blobColor) {
+  this.name = blobName;
+  this.color = blobColor;
+  this.rateOfConsumption = 1;
+  this.peopleConsumed = 0;
+  this.elapsedTime = 0;
+}
 
- TODO: Next, create an instance of Blob named blob.
+var blob = new Blob("Blob", "Blue");
 
- TODO: Then, use a loop to calculate how long it took the blob to finish
- with Dowington.
-*/
+var hoursSpentInDowington;
+
+while (blob.peopleConsumed < 1000) {
+  blob.peopleConsumed += 1;
+  blob.elapsedTime += (1 / blob.rateOfConsumption);
```

Comment:
You've got some seriously crazy blob growth going on here. If you mimic the rate that was in the table, you wouldn't need to divide by rateOfConsumption here. No points off, though, since I know the wording was confusing to some.